cognite/client/_api/files.py

```diff
@@ -229,20 +240,24 @@ def list(
                 >>> for file_list in c.files(chunk_size=2500):
                 ...     file_list # do something with the files
         """
+        if (root_asset_ids and not isinstance(root_asset_ids[0], dict)) or root_asset_external_ids:
```

Comment:
Don't need the backwards compatiblity on root_asset_ids being `List[Dict]` here, since we never supported it.